packages/framework/src/Component/Error/ErrorPageCronModule.php

```diff
@@ -1,30 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\FrameworkBundle\Component\Error;
-
-use Monolog\Logger;
-use Shopsys\Plugin\Cron\SimpleCronModuleInterface;
-
-class ErrorPageCronModule implements SimpleCronModuleInterface
```

Comment:
should we write the migration to remove the cron module from the database?

Reply:
yes this would be nice ... fixed